src/main/webapp/app/shared/metis/posting-content/posting-content.component.html

```diff
@@ -9,10 +9,14 @@
         </span>
         <div *ngIf="showContent">
             <jhi-posting-content-part *ngFor="let postingContentPart of postingContentParts" [postingContentPart]="postingContentPart"> </jhi-posting-content-part>
+
+            <span class="edited-text" *ngIf="isEdited">(edited)</span>
```

Comment:
shouldn't you use the translated text here as well?

Reply:
Good catch! I updated it.